setup.cfg

```diff
@@ -5,4 +5,5 @@ universal = 1
 license_file = LICENSE
 
 [tool:pytest]
-xfail_strict = true
\ No newline at end of file
+xfail_strict = true
+doctest_optionflags = ALLOW_UNICODE ELLIPSIS
```

Comment:
I think that if "NORMALIZE_WHITESPACE" is added then some long lines can be wrapped.

There's also an option "addopts = --doctest-modules" that I've tried but that has broken for me on non-Windows by trying to import tzwin.  It'd be nice if that was somehow available.